src/unix/bsd/freebsdlike/freebsd/freebsd11/mod.rs

```diff
@@ -31,99 +31,182 @@ s! {
         pub ki_tsize: ::segsz_t,
+        /// Data size (pages) XXX.
         pub ki_dsize: ::segsz_t,
+        /// Stack size (pages).
         pub ki_ssize: ::segsz_t,
+        /// Exit status for wait & stop signal.
         pub ki_xstat: ::u_short,
+        /// Accounting flags.
         pub ki_acflag: ::u_short,
+        /// %cpu for process during `ki_swtime`.
         pub ki_pctcpu: ::fixpt_t,
+        /// Time averaged value of `ki_cpticks`.
         pub ki_estcpu: ::u_int,
+        /// Time since last blocked.
         pub ki_slptime: ::u_int,
+        /// Time swapped in or out.
         pub ki_swtime: ::u_int,
+        /// Number of copy-on-write faults.
         pub ki_cow: ::u_int,
+        /// Real time in microsec.
         pub ki_runtime: u64,
+        /// Starting time.
         pub ki_start: ::timeval,
+        /// Time used by process children.
         pub ki_childtime: ::timeval,
+        /// P_* flags.
         pub ki_flag: ::c_long,
+        /// KI_* flags (below).
         pub ki_kiflag: ::c_long,
+        /// Kernel trace points.
         pub ki_traceflag: ::c_int,
+        /// S* process status.
         pub ki_stat: ::c_char,
+        /// Process "nice" value.
         pub ki_nice: i8, // signed char
+        /// Process lock (prevent swap) count.
         pub ki_lock: ::c_char,
+        /// Run queue index.
         pub ki_rqindex: ::c_char,
+        /// Which cpu we are on.
         pub ki_oncpu_old: ::c_uchar,
+        /// Last cpu we were on.
         pub ki_lastcpu_old: ::c_uchar,
+        /// Thread name.
         pub ki_tdname: [::c_char; ::TDNAMLEN + 1],
+        /// Wchan message.
         pub ki_wmesg: [::c_char; ::WMESGLEN + 1],
+        /// Setlogin name.
         pub ki_login: [::c_char; ::LOGNAMELEN + 1],
+        /// Lock name.
         pub ki_lockname: [::c_char; ::LOCKNAMELEN + 1],
+        /// Command name.
         pub ki_comm: [::c_char; ::COMMLEN + 1],
+        /// Emulation name.
         pub ki_emul: [::c_char; ::KI_EMULNAMELEN + 1],
+        /// Login class.
         pub ki_loginclass: [::c_char; ::LOGINCLASSLEN + 1],
+        /// More thread name.
         pub ki_moretdname: [::c_char; ::MAXCOMLEN - ::TDNAMLEN + 1],
+        /// Spare string space.
         pub ki_sparestrings: [[::c_char; 23]; 2], // little hack to allow PartialEq
+        /// Spare room for growth.
         pub ki_spareints: [::c_int; ::KI_NSPARE_INT],
+        /// Which cpu we are on.
         pub ki_oncpu: ::c_int,
+        /// Last cpu we were on.
         pub ki_lastcpu: ::c_int,
+        /// PID of tracing process.
         pub ki_tracer: ::c_int,
+        /// P2_* flags.
         pub ki_flag2: ::c_int,
+        /// Default FIB number.
         pub ki_fibnum: ::c_int,
+        /// Credential flags.
         pub ki_cr_flags: ::u_int,
+        /// Process jail ID.
         pub ki_jid: ::c_int,
+        /// Number of threads in total.
         pub ki_numthreads: ::c_int,
+        // Thread ID.
```

Comment:
Shouldn't this be a doc comment? (Ditto for other versions of this struct)

Reply:
Yes it should. Wanna send a PR?

Reply:
Sure, here it is: https://github.com/rust-lang/libc/pull/2563

Reply:
Thanks!